### ISSUE TITLE
Chaos draft with all the sets appearing ~equally

### ISSUE THREAD

**Issue by sprangg** [Senryoku/MTGADraft#350]
Hello! While mtgadraft works for chaos drafting with the "random set from card pool" boosters, because of the truly randomly pulled packs, the resulting pool can exclude one or more of the chosen sets and include multiples of another, even when the number of chosen sets is <= the number of packs opened in the draft.

Would it be possible to either modify the current "random set from card pool" or add another similar option, so that it'd be possible to do a chaos draft where all the chosen sets are guaranteed to appear if the number of chosen sets is <= the number of packs opened in the draft?

Basically this means that 2nd pack of a set isn't opened until all the sets have been opened once. Same goes for the 3rd pack of a set, not opened until all the sets have been opened twice, etc...

**Comment by Senryoku** [comment]
Hi!

I don't want to add an option for such a small change, but I think it makes sense as the default.
I made the change here c81f8492076971a282a178cee13f18810e547e22 and it will be part of the next deployment.

**Comment by sprangg** [comment]
Rock 'n' roll! Thank you!
